webpack.common.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
 				test: /\.vue$/,
 				loader: 'vue-loader',
 				exclude: babelLoaderExcludeNodeModulesExcept([
-					'vue-material-design-icons',
+					'vue-material-design-icons', '@nextcloud/vue',
```

Comment:
since we do it in the components already, if you're facing a bug, this needs to be addressed there.

Also: I doubt you are really importing `vue` files from `@nextcloud/vue` :thinking:

Reply:
This was needed in order to use MD icons in the vue library. It was @juliushaertl suggestion. I'm not using icons there anymore since I'm doing a description component in talk. Should I remove it?

Reply:
> I'm not using icons there anymore since I'm doing a description component in talk. Should I remove it?

yes :+1: